text/0000-core-subteam.md

```diff
@@ -0,0 +1,107 @@
+- Feature Name: core_subteam
+- Start Date: 2019-05-04
+- RFC PR:
+- Hyperledger Grid Issue:
+
+# Summary
+[summary]: #summary
+
+Create a new Grid Core subteam to shepherd technical RFCs through the RFC
+process, and make technical governance decisions; technical topics include
+those related to the design and architecture of Grid. This team is responsible
+for ensuring long-term architectural consistency with the existing Grid code
+and overall project direction adopted by the Grid Root team.
+
+The Grid Core subteam shall be the default subteam for technical decisions in
+absence of another appropriate subteam.
+
+# Motivation
+[motivation]: #motivation
+
+[Grid RFC 0008](https://github.com/hyperledger/grid-rfcs/blob/master/text/0008-grid-governance.md)
```

Comment:
This link is dead - either filename needs to be changed or something.  It's not clear from this section in docs if it should have been changed to `0008` or left as `0000`.

> Copy 0000-template.md to text/0000-my-feature.md (where "my-feature" is descriptive). Don't assign an RFC number yet (leave 0000 in the name).

Reply:
Fixed via another PR.